diff --git a/howdy/src/meson.build b/howdy/src/meson.build
index 36dc05e..c624eaf 100644
--- a/howdy/src/meson.build
+++ b/howdy/src/meson.build
@@ -101,10 +101,7 @@ install_data(
     rename: 'howdy',
 )
 
-fs = import('fs')
-if not fs.exists(config_path)
-    install_data('config.ini', install_dir: confdir, install_mode: 'rwxr--r--', install_tag: 'config')
-endif
+install_data('config.ini', install_dir: confdir, install_mode: 'rwxr--r--', install_tag: 'config')
 
 install_data('dlib-data/install.sh', 'dlib-data/Readme.md', install_dir: dlibdatadir, install_mode: 'rwxr--r--')
 
